main.js

```diff
@@ -0,0 +1,98 @@
+///////////////////////////////////////Variables
+
+var i = 0; i < 10;
+// var connect3 = (["", "", ""] || ["", "", ""] || ["", "", ""]);
+
+///////////////////////////////////////Game Controller
+
+function turnAlternator(){
+  changeTurn[i] = document.addEventListener('click', function(event){
+    i++
+
+  })
+}
+
+// $(function({
+//   $('td').on('click', function(){
+//     i++
+//   })
+//   })
+// })
+
+function changeTurn(){
```

Comment:
If you want this to be the function that runs when something is clicked, you would write it like this:

$('td').click(changeTurn)

i.e., when td is clicked, it calls this function. Right now you are defining the event listener within this function, but the function is never being called.